### ISSUE TITLE
Transition from include -> extend

### ISSUE THREAD

**Issue by adomokos** [adomokos/light-service#63]
Right now you can mix-in an organizer or action specific logic by "including" the module, like this:

```ruby
class SomeOrganizer
  include LightService::Organizer

  def self.for(user)
  end
end
```

This is silly, the logic mixed into the class will be at class level, and the default for that in Ruby is "extend". We should phase out "include" with a warning and just use "extend" going forward.

Thoughts?

**Comment by padi** [comment]
As far as I can remember, extending still works. https://github.com/adomokos/light-service/pull/10

It just doesn't have the proper spec to catch this when it fails (since I replaced the spec to use `include`). So better make sure `extend` still works.

As long as you can make Organizer-Actions, this seems fine to me.

**Comment by adomokos** [comment]
Thank you for pointing out that PR. I am thinking about keeping `include`, but it will give warning when that's used. And eventually I'd phase it out.

**Comment by padi** [comment]
^ Done.

**Comment by adomokos** [comment]
Whenever you are in Chicago, IL, I'll buy you lunch or dinner. :heart: Thank you!

**Comment by adomokos** [comment]
PR merged, will be released with version 0.6.
